README.md

```diff
@@ -45,6 +45,26 @@ using (var audit = new SerilogSeqAuditClient(
 }
 ```
 
+If you wish to customise the behaviour audit logger, you may do so:
```

Comment:
Should we mention here that this should only be used by people who know what they're doing and are willing to take on the extra risk? It's easy to screw things up, and we (Logic Audit) may not be able to help you.

Reply:
I reworded the intro to be quite explicit.